frontend/src/metabase/ui/components/data-display/Image/index.tsx

```diff
@@ -0,0 +1,16 @@
+import type { ImageProps as MantineImageProps } from "@mantine/core";
+import { Image as MantineImage } from "@mantine/core";
+import type { CSSProperties } from "react";
+
+export interface ImageProps extends MantineImageProps {
+  position: CSSProperties["position"];
+}
+
+export function Image({ position, ...mantineImageProps }: ImageProps) {
+  return (
+    <MantineImage
+      {...mantineImageProps}
+      styles={{ image: { objectPosition: position } }}
```

Comment:
I found it weird that `Image` [only support `fit` for `object-fit` but not `postion` for `object-position`.](https://v6.mantine.dev/core/image/?t=props)

This adds support for `position`, since I don't think it's too hard to use `styles={{ image: { objectPosition: position } }}` whenever we wants to adjust `object-position`